spec/services/rapid_ready_for_decision/fast_track_pdf_generator_spec.rb

```diff
@@ -10,7 +10,7 @@
 
   if ENV['_SAVE_RRD_PDF_FILE']
     after do |example|
-      file_name = "tmp/rrd-pdf-preview-#{example.metadata[:description].parameterize}-#{Time.now.to_i}.pdf"
+      file_name = "tmp/rrd-pdf-preview-#{example.full_description.parameterize}-#{Time.now.to_i}.pdf"
```

Comment:
Was wondering about full_description here but got cleared up by Matt, sounds like it's an rspec method.